lib/acmesmith/post_issueing_hooks/shell.rb

```diff
@@ -0,0 +1,23 @@
+require 'erb'
+require 'acmesmith/post_issueing_hooks/base'
+
+module Acmesmith
+  module PostIssueingHooks
+    class Shell < Base
+      class HostedZoneNotFound < StandardError; end
+      class AmbiguousHostedZones < StandardError; end
+
+      def initialize(common_name:, command:)
+        @common_name = common_name
+        @command = command
+      end
+
+      def execute()
+        parsed_command = ERB.new(@command).result(binding)
+        puts "=> Executing Post Issueing Hook for #{@common_name} in #{self.class.name}"
+        puts "=> Running #{parsed_command}"
+        system(parsed_command)
```

Comment:
I now add a configurable error handling. By default exit > 0 raises a fatal, but a command can have the ignore_failure set true. Borrowed this from Chef. Do you like it this way? Config is in ```config.mock.yml```